lib/stat_tracker.rb

```diff
@@ -25,4 +25,60 @@ def from_csv(locations_hash)
       @game_by_team << Game_By_Team.new(row)
     end
   end
+      
+#---------Game Statics Methods-----------
+  def percentage_ties
+    tie_count = @games.count { |game| game.away_goals.to_i == game.home_goals.to_i }
+    percentage = (tie_count.to_f / @games.count.to_f).round(4) * 100
+    p percentage
+  end
+
+  def count_of_games_by_season
+  season_games = @games.each_with_object(Hash.new(0)) {|game, hash| hash[game.season] += 1}
+  p season_games
+  end
+
+  def highest_total_score
+    highest_score = 0
+    @games.each do |game|
+      # binding.pry
+      total_score = game.home_goals.to_i + game.away_goals.to_i
+      highest_score = total_score if total_score > highest_score
+    end
+    highest_score
+  end
+
+  def lowest_total_score
+    lowest_score = nil
+    @games.each do |game|
+      total_score = game.home_goals.to_i + game.away_goals.to_i
+      lowest_score = total_score if lowest_score.nil? || total_score < lowest_score
+    end
+    lowest_score
+  end
+
+  def percentage_visitor_wins
```

Comment:
can make percentage home_wins and percentage away_wins look more similar when we refactor as they are basically the same process